javascript-modal/main.js

```diff
@@ -0,0 +1,12 @@
+var modal = document.querySelector('.modal');
+
+var modalOpenButton = document.querySelector('.open');
+var modalCloseButton = document.querySelector('.modal-close');
+
+modalOpenButton.onclick = function () {
+  modal.style.display = 'flex';
+};
+
+modalCloseButton.onclick = function () {
+  modal.style.display = 'none';
```

Comment:
This will end up inline styling on the element, which should be avoided, for reasons of specificity. You should prepare a class for each of these, and swap the classes with JavaScript